pkg/new-ui/v1beta1/frontend/src/app/models/experiment.model.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * UI relative types
+ */
+interface ExperimentCondition {
+  type: string;
+  status: string;
+  reason: string;
+  message: string;
+  lastUpdateTime: string;
+  lastTransitionTime: string;
+}
+
+export interface TrialParameterAssignments {
+  name: string;
+  value: string;
+}
+
+export interface TrialObservationMetrics {
+  name: string;
+  min: number;
+  max: number;
+  latest: number;
+}
+
+interface ExperimentCurrentOptimalTrial {
+  bestTrialName: string;
+  parameterAssignments: TrialParameterAssignments[];
+  observation: {
+    metrics: TrialObservationMetrics[];
+  };
+}
+
+export interface Experiment {
+  Name: string;
+  Namespace: string;
+  Status: string;
+  startTime: string;
+  conditions: ExperimentCondition[];
+  currentOptimalTrial: ExperimentCurrentOptimalTrial;
+  runningTrialList: string[];
+  failedTrialList: string[];
+  succeededTrialList: string[];
+  trials: number;
+  trialsSucceeded: number;
+  trialsFailed: number;
+  trialsRunning: number;
```

Comment:
We might include other Experiment status parameters, e.g. https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1beta1/experiment_types.go#L121.
I think we can done it later.

Reply:
ACK, I've created an issue to keep track of this https://github.com/kubeflow/katib/issues/1434

I'll also use the `[rev]` prefix for the issues we create during the review process so we can easily grep them in the future. If you don't like this convention I can remove this prefix

Reply:
It's fine, thank you @kimwnasptd!